lib/experimental/Information/Shortcut/index.tsx

```diff
@@ -37,12 +37,17 @@ function Shortcut({ keys, variant }: ShortcutProps) {
         return (
           <kbd
             key={index}
+            role="term"
             className={cn(
               shortcutVariants({ variant }),
               isIcon ? "w-5 px-0.5" : "min-w-5 px-1"
             )}
           >
-            {isIcon ? <Icon icon={iconMap[lowerKey]} size="sm" /> : key}
+            {isIcon ? (
+              <Icon data-testid="cmd-icon" icon={iconMap[lowerKey]} size="sm" />
```

Comment:
I'd prefer to avoid using `data-testid` where possible. Do you think we can avoid it here?